app/lib/google_scraper_ruby/errors.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module GoogleScraperRuby
+  module Errors
+    class SearchServiceError < StandardError
+      def initialize(url:, error: nil)
+        @original_error = error
+        @url = url
+        super('')
```

Comment:
Can we just send the original error to the parent 
```suggestion
        super(error)
```